src/controller/python/test_client/rpc_client.py

```diff
@@ -1,6 +1,7 @@
-import xmlrpc.client
+from xmlrpc.client import ServerProxy
 import docker
 import time
+import xmlrpc
```

Comment:
Why do we need import:
`from xmlrpc.client import ServerProxy` and  `import xmlrpc`, even if in the code ServerProxy object is created by 
`server_1 = xmlrpc.client.ServerProxy("http://localhost:5050/")`